README.md

```diff
@@ -190,6 +190,7 @@ Every dataset and pipeline folder must contain a `dataset.yaml` and a `pipeline.
 
 # Best Practices
 
+- When running `scripts/generate_terraform.py`, the argument `--bucket-name-prefix` helps prevent GCS bucket name collisions because bucket names must be globally unique. Make the prefix as unique as possible, and specific to your own environment or use case.
```

Comment:
There needs to be some mention about this in step 3 of the instructions as well

Reply:
Good catch! Please see 8c27336.